File: swimming-contest-software/Model/src/main/java/protocol/responses/FindAllRacesDetailsResponse.java
package protocol.responses;

import domain.dtos.RaceDTO;

import java.util.List;

public class FindAllRacesDetailsResponse implements Response {
    private final List<RaceDTO> allRacesDetails;

    public FindAllRacesDetailsResponse(List<RaceDTO> allRacesDetails) {
        this.allRacesDetails = allRacesDetails;
    }

    public List<RaceDTO> getAllRacesDetails() {
        return allRacesDetails;
    }
}

File: Model/Protocol/Responses/FindAllRacesDetailsResponse.cs
using System;
using System.Collections.Generic;
using Model.Domain.DTOs;

namespace Model.Protocol.Responses;

[Serializable]
public class FindAllRacesDetailsResponse : IResponse
{
    public FindAllRacesDetailsResponse(List<RaceDTO> allRacesDetails)
    {
        AllRacesDetails = allRacesDetails;
    }

    public List<RaceDTO> AllRacesDetails { get; }
}